lib/read.js

```diff
@@ -24,8 +24,21 @@ var Read = Juttle.proc.source.extend({
             throw new Error('Unknown option ' + unknown[0]);
         }
 
-        if (options.from && options.to && JuttleMoment.gt(options.from, options.to)) {
-            throw new Error('From cannot be after to');
+        if (options.raw) {
+            if (options.from || options.to) {
+                throw new Error('-raw option should not be combined with -from, -to, or -last');
+            }
+        } else {
+            if (!options.from && !options.to) {
+                throw new Error('One of -from, -to, or -last must be specified to define a query time range');
+            }
+
+            options.from = options.from || program.now;
+            options.to = options.to || program.now;
+
+            if (JuttleMoment.gt(options.from, options.to)) {
```

Comment:
This check is unnecessary. See https://github.com/juttle/juttle/blob/master/lib/runtime/procs/source.js#L57